common/vm_get_primary_nic.yml

```diff
@@ -16,18 +16,19 @@
   ansible.builtin.assert:
     that:
       - vm_guest_mac_ipv4 | length >= 1
+      - vm_guest_mac_ipv4.values() | select | length >= 1
```

Comment:
What's this condition for?

Reply:
Check there is IPv4 address in the mac-ip dictionary.